functions.php

```diff
@@ -4,6 +4,7 @@
  *
  * @link https://developer.wordpress.org/themes/basics/theme-functions/
  *
- * @package Twenty Twenty-Five
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
  * @since Twenty Twenty-Five 1.0
  */
```

Comment:
This change is unrelated, but updates the packages consistently after https://github.com/WordPress/twentytwentyfive/pull/18